pom.xml

```diff
@@ -63,5 +64,4 @@
             </plugins>
         </pluginManagement>
     </build>
-</project>
-
+</project>
```

Comment:
why do you need to change pom.xml file?

Reply:
I ve got problems with mvn package check